### ISSUE TITLE
Adjustable ScaleX Precision/Detail

### ISSUE THREAD

**Issue by ScriptTiger** [hvianna/audioMotion-analyzer#76]
Is there any way that the precision/detail of the ScaleX could be adjustable to make it more precise/detailed? For example, if a frequency range of 20 kHz to 22 kHz is used in order to zoom in on the ultrasonics, currently the most precise/detailed mode is linear, but the markers jump from 20 kHz to 22 kHz and does not display anything in between. Is there a way to increase the frequency of markers to, for example, every 100 Hz, instead of every 2 KHz, when linear mode is being used?

Also, I know that 20 kHz to 22 kHz is not a range currently available in the tool, as the lowest lower limit is currently 16 Hz. However, modifying that is not an issue. I'm just having a problem modifying the precision/detail of the ScaleX.

**Comment by hvianna** [comment]
Yes, I need to come up with a generic solution that's good enough for different ranges.
